.github/workflows/poll-email-processing.yml

```diff
@@ -0,0 +1,14 @@
+name: Poll Email Processing
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '* * * * *'
```

Comment:
If you want to run this every 5 minutes you can do something like the below:

`- cron: '*/5 * * * *'`

The cron schedule is set to run every minute currently.

Reply:
It is set to 1 minute intentionally, I believe this is what we discussed as a team and is what I've done the performance analysis within the ticket - see analysis details at bottom of ticket.

Although I will be discussing with the team if this is the best approach (or if we should use an Azure function) anyway and if not we can change/remove this workflow.